lld/MachO/Driver.cpp

```diff
@@ -1709,7 +1709,7 @@ bool link(ArrayRef<const char *> argsArr, llvm::raw_ostream &stdoutOS,
 
     firstTLVDataSection = nullptr;
     tar = nullptr;
-    memset(&in, 0, sizeof(in));
+    in = InStruct();
```

Comment:
I think adding a vector and map to this struct made it no longer trivially copyable. This seems like a better way to clear it anyway.